openapi-core/src/main/java/io/ballerina/openapi/core/GeneratorUtils.java

```diff
@@ -1014,4 +1020,23 @@ private static SemanticModel getSemanticModel(Path clientPath) throws ProjectExc
             throw new ProjectException(e.getMessage());
         }
     }
+
+    /**
+     * This utility is used to select the Ballerina data type for a given numeric format type.
+     *
+     * @param dataType name of the data type. ex: number, integer
+     * @param schema uses to generate the type descriptor name ex: int32, int64
+     * @return data type for invalid numeric data formats
+     */
+    public static String convertOpenAPINumericTypeToBallerina(final String dataType, final Schema<?> schema) {
+        try {
+            if (schema.getFormat() != null) {
+                return GeneratorUtils.convertOpenAPITypeToBallerina(schema.getFormat().trim());
+            }
+        } catch (BallerinaOpenApiException e) {
+            OUT_STREAM.printf("WARNING: unsupported format `%s` will be skipped when generating the counterpart " +
+                    "Ballerina type for openAPI schema type: `%s`", schema.getFormat(), schema.getType());
```

Comment:
```suggestion
            OUT_STREAM.printf("WARNING: unsupported format `%s` will be skipped when generating the counterpart " +
                    "ballerina type for openapi schema type: `%s`", schema.getFormat(), schema.getType());
```

Reply:
I don't think we lowercase the whole message. @praneesha WDYT?

Reply:
Industrial terms such as Ballerina and OpenAPI have to be added in their respective standard formats.

Reply:
As discussed at the meeting here I suggested to used `ballerina` for make consist with the Ballerina lang and other tooling [references](https://github.com/ballerina-platform/module-ballerina-graphql/blob/993d54980ddbc42843e39e668b821de91dc8f886/compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/diagnostics/DiagnosticMessage.java#L34) errors/warnings

Reply:
But we seem to have used `Ballerina` in the other openAPI diagnostics as well. https://github.com/ballerina-platform/openapi-tools/blob/master/openapi-bal-service/src/main/java/io/ballerina/openapi/converter/diagnostic/DiagnosticMessages.java
So I prefer keeping it as is.

Reply:
+1 for change